python/xorq/caching/__init__.py

```diff
@@ -21,7 +21,7 @@
 from opentelemetry import trace
 from public import public
 
-import xorq as xo
+import xorq.api as xo
```

Comment:
its okay to use `xorq.api` in tests and examples but we shouldn't be using it within our own library code